utils/serialization_utils.js

```diff
@@ -30,22 +30,22 @@ const Transform = require('stream').Transform;
 function DeserializeStream(options) {
   if (!(this instanceof DeserializeStream)) {
     return new DeserializeStream(options);
-	}
+  }
 
   Transform.call(this, options);
-	// true once we've pulled off the message length
-	// for the next message we'll need to deserialize
+  // true once we've pulled off the message length
+  // for the next message we'll need to deserialize
   this._inBody = false;
 
-	// track how many bytes of this message we've received so far
-	this._messageConsumed = 0;
+  // track how many bytes of this message we've received so far
+  this._messageConsumed = 0;
 
-	// how long this message will be
-	this._messageLen = -1;
+  // how long this message will be
+  this._messageLen = -1;
 
-	// as bytes of this message arrive, store them in this
-	// buffer until we have the whole thing
-	this._messageBuffer = [];
+  // as bytes of this message arrive, store them in this
+  // buffer until we have the whole thing
+  this._messageBuffer = [];
 
   // FIXME: These are specific to parsing a service response...
```

Comment:
FIXME here is maybe misleading... its more like a TODO. DeserializeStream is being used for any TCPROS deserialization, but the scheme is slightly different for services than for everything else so this is mostly me saying "Maybe don't conflate these things"